ops/stg/api.tf

```diff
@@ -7,6 +7,8 @@ module "simple_report_api" {
   instance_tier  = "PremiumV3"
   instance_size  = "P2v3"
 
+  sku_name = "P2v3"
```

Comment:
does this replace `instance_size`/`instance_tier`?

Reply:
It does look like these variables are no longer used and can be removed. I'll go ahead and do that in this PR.